sale_secondary_salesperson/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Secondary Salesperson',
```

Comment:
Please change all these simple quotes to double quotes for avoiding a big change when migrating to 13.0